pom.xml

```diff
@@ -147,6 +152,12 @@
             <version>${logback.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>31.1-jre</version>
+        </dependency>
```

Comment:
why is this needed? I haven't seen any usage in this PR

Reply:
It was needed because of this error:
![image](https://user-images.githubusercontent.com/106826306/192529164-0d665fd6-3159-44fb-ad2f-348b18724a8e.png)

The google-http-client library was calling this method from guava. But guava is a library which many other libraries are using therefore it was imported from multiple places with different versions. The version used in our projects was 16 which did not implement that method. That method existed in guava only from version 20. Therefore first i thought to just add the guava library itself with the latest version. However this did not work in eclipse plugin. So the final working solution was to increase the version of guice library to 4.2.0 which brings guava 23.6